Modules/Filtering/FFT/itk-module.cmake

```diff
@@ -14,6 +14,8 @@ itk_module(ITKFFT
   TEST_DEPENDS
     ITKTestKernel
     ITKImageCompare
+  FACTORY_NAMES
+    FFT::VnlForward1D
```

Comment:
Note the syntax defined here generally mirrors `FACTORY_NAMES` syntax for IO classes, with minor corrections happening in `UseITK.cmake`. Namely, `FFT::<fft-type>` maps to a `<fft-type>FFTImageFilterFactoryRegister__Private()` function header in `itkFFTImageFilterFactoryRegisterManager.h` which must be implemented in a `.cxx` source file.